ts/Stock/Indicators/VBP/VBPIndicator.ts

```diff
@@ -829,10 +829,7 @@ interface VBPIndicator {
 extend(VBPIndicator.prototype, {
     nameBase: 'Volume by Price',
     nameComponents: ['ranges'],
-    bindTo: {
-        series: false,
-        eventName: 'afterSetExtremes'
-    },
+    calculateOnExtremesChagne: true,
```

Comment:
Change*

Would it make sense to merge somehow both properties: `calculateOn` and `calculateOnExtremesChange`?

Reply:
I combined them but I'm not sure if this is a correct approach.
Each one is responsible for something else and the main issue that I have with it is finding a proper name for this combined property...